src/test/mir-opt/const_prop/read_immutable_static.rs

```diff
@@ -10,10 +10,10 @@ fn main() {
 // START rustc.main.ConstProp.before.mir
 //  bb0: {
 //      ...
-//      _3 = const Scalar(AllocId(0).0x0) : &u8;
+//      _3 = const AllocId(0).0x0;
 //      _2 = (*_3);
 //      ...
-//      _5 = const Scalar(AllocId(0).0x0) : &u8;
+//      _5 = const AllocId(0).0x0;
```

Comment:
This syntax irks me a bit. Could we do something more like `alloc0+0x0` or thereabouts?

Also, why is the type missing? Maybe `const &{*(alloc0+0): u8}` would make some sense?

Reply:
I opted for `{alloc0+0: &u8}`